contracts/dtoken/tests/userflows/userflow.liquidation.feature

```diff
@@ -0,0 +1,47 @@
+Feature: User liquidation flow
+
+	Rule: Price for WETH, WNEAR are should be the same 5$
+		Background:
+			Given The borrower, liquidator users, WETH, WNEAR, DWETH, DWNEAR contracts
+			And Digital token DWETH contract with supplied 100 tokens by borrower
+			And Digital token DWNEAR contract with supplied 50 tokens by borrower
+			And Digital token DWETH contract with supplied 100 tokens by liquidator
+			And health_threshold should be equal 150% === 15000 in Ratio format
+			And Contracts accrued interests should be equal 0
+			And token = 10^24
+
+
+		Scenario: Liquidator tries to liquidate Borrower - negative flow  <Borrower HF in good condition>
+			Given Borrower, Liquidator, WETH, WNEAR, DWETH, DWNEAR contracts
+			When Liquidator wants to liquidate 50 WNEAR with 50 DWETH
```

Comment:
@pahomas I suppose we should not provide revenue here (either test it in Then or don't test in this scenario at all)

Reply:
Let's discuss